sormas-backend/src/main/java/de/symeda/sormas/backend/common/BaseAdoService.java

```diff
@@ -165,6 +169,16 @@ public Timestamp getLatestChangeDate() {
 		return em.createQuery(cq).getResultList();
 	}
 
+	public List<ADO> getBatchedQueryResults(CriteriaBuilder cb, CriteriaQuery<ADO> cq, From<?, ADO> from, Integer batchSize) {
+
+		if (batchSize != null) {
+			cq.orderBy(cb.asc(from.get(AbstractDomainObject.CHANGE_DATE)), cb.asc(from.get(AbstractDomainObject.UUID)));
+		} else {
+			cq.orderBy(cb.desc(from.get(AbstractDomainObject.CHANGE_DATE)));
```

Comment:
Please also change this to ascending. I'm aware that this will change the old behaviour, but the old behaviour was acutally wrong and only worked because we have always synced the whole list.

Reply:
done